lldbinit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Initialization for lldb.
```

Comment:
Phrase as a docstring?

```suggestion
"""Initialization for lldb."""
```

Reply:
Done